.github/workflows/publish-artefact.yaml

```diff
@@ -0,0 +1,84 @@
+name: "Integration Tests and Publish on Tag"
+
+on:
+  push:
+    tags:
+    - 'autonity-v[0-9]+.[0-9]+.[0-9]+*'
+    - 'audonity-demo-v[0-9]+.[0-9]+.[0-9]+*'
+    - 'autonity-network-v[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
+  integration_tests_publish_on_tag:
+    runs-on: ubuntu-latest
```

Comment:
Could we use latest `ubuntu-20.04` instead?

Reply:
Done